templates/bibrec.html

```diff
@@ -291,7 +291,7 @@ <h2 id="about_book_header">About this eBook <span>${help (_('Table: Bibliographi
 			<td py:when="marc.code[0]=='5'">
 			  <?python
 			    text = gg.xmlspecialchars(marc.text)
-			    text = re.sub(r'(//\S+)', r'<a href="\1">\1</a>', text)
+			    text = re.sub(r'((http:)?(https:)?(//\S+))', r'<a href="\1">\1</a>', text)
```

Comment:
Probably not a big deal, but I think the current regex would match `http:https://www.wikipedia.org/`.
```suggestion
			    text = re.sub(r'(http:|https:)?(//\S+))', r'<a href="\1">\1</a>', text)
```